### PR TITLE
prov/sockets: Added description of env var in the manpage

### PR DESCRIPTION
**Pull request by shantonu**
- Added a section to describe available environment variables.
- Added FI_SOCKETS_PE_AFFINITY in SOCKETS_INI
- Added a function to read the default env parameters

@jithinjosepkl, can you please review?

Signed-off-by: Shantonu Hossain <shantonu.hossain@intel.com>

### CODE REVIEW

File: man/fi_sockets.7.md [hunk 1]
```diff
@@ -51,6 +51,31 @@ performance numbers are lower compared to other providers implemented
...
+
+*FI_SOCKETS_DEF_CQ_SZ*
+: An integer to specify the default completion queue size.
+
+*FI_SOCKETS_DEF_EQ_SZ*
+: An integer to specify the default event queue size.
+
+*FI_SOCKETS_DGRAM_DROP_RATE*
+: An integer value to specify the drop rate of dgram frame when endpoint is *FI_EP_DGRAM*. This is for debugging purpose only.
+
```
Comment (jithinjosepkl) [inline man/fi_sockets.7.md]: We should add FI_SOCKETS_PE_AFFINITY as well.

File: man/fi_sockets.7.md [hunk 2]
```diff
@@ -51,6 +51,31 @@ performance numbers are lower compared to other providers implemented
 over high-speed fabric, and lower than what an application might see
 implementing to sockets directly.
 
+# RUNTIME PARAMETERS
+
+The sockets provider checks for the following environment variables. For scalability testing a user can use these environment variables to tweak the default sizes of the address vector(AV), completion queue (CQ), connection map etc. that satisfies the requriment of the particular benchmark.
+
```
Comment (jithinjosepkl) [inline man/fi_sockets.7.md]: Instead, should we mention this as a side note (recommended parameters for large scale runs) ?

**Comment by jithinjosepkl** [comment]
Thanks @shantonu. I think the FI_SOCKETS_PE_AFFINITY will not show up if we do fi_info -e, since this parameter is not defined in SOCKETS_INI.

@shefty - Should we move all fi_param_define calls into SOCKETS_INI ? Asking because sometime earlier you had mentioned its better to define the parameters just before they are used.

File: man/fi_sockets.7.md (hunk 2 above)
Comment (shantonu) [inline man/fi_sockets.7.md]: :+1:

**Comment by shefty** [comment]
I think it's better to *read* the parameter values just before they are used.

**Comment by jithinjosepkl** [comment]
@shefty - Thanks. yes, makes sense.

@shantonu - Can you please update the PR accordingly?

**Comment by shantonu** [comment]
Updated the PR and added a function to read the env parameters. Can you please review @jithinjosepkl ?

**Pull request merged**